index.html

```diff
@@ -0,0 +1,1224 @@
+
+                  <div class="note" id="subject-verification" inlist="" rel="schema:hasPart" resource="#subject-verification">
+                    <h3 property="schema:name"><span>Note</span>: Subject Verification</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>Servers may accept different authentication protocols as well as credential verification methods.</p>
+                    </div>
+                  </div>
+
+                  <div class="note" id="origin-considerations" inlist="" rel="schema:hasPart" resource="#origin-considerations">
+                    <h3 property="schema:name"><span>Note</span>: Origin Considerations</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>Typical Web browsers use origin based security. While it has many limitations, it is however ubiquitous.</p>
+                    </div>
+                  </div>
+
+                  <div class="issue" id="client-identification" rel="schema:hasPart" resource="#client-identification">
+                    <h3 property="schema:name"><span>Issue</span>: Client Identification</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>Distinguishing social entities and clients: <a href="https://github.com/solid/web-access-control-spec/issues/81" rel="cito:citesAsRelated">issues/81</a>.</p>
+                    </div>
+                  </div>
+
+                  <p class="advisement">Source: <a href="https://github.com/solid/web-access-control-spec/issues/34" rel="cito:citesAsSourceDocument">issues/34</a>, <a href="https://github.com/solid/specification/issues/32" rel="cito:citesAsSourceDocument">issues/32</a>, <a href="https://github.com/solid/specification/issues/59">issues/59</a></p>
+                </div>
+              </section>
+            </div>
+          </section>
+
+          <section id="authorization-process" inlist="" rel="schema:hasPart" resource="#authorization-process">
+            <h2 property="schema:name">Authorization Process</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <p>This section describes how implementations can determine the <a href="#effective-acl-resource" rel="cito:discusses">effective ACL resource</a> of a resource and hence which set of <a href="#authorization-conformance" rel="cito:discusses">conforming authorizations</a> to use.</p>
+
+              <p>Servers process access requests by <a href="#authorization-evaluation" rel="cito:discusses">evaluating the Authorizations</a> associated with referenced resources in order to determine whether the necessary access permissions are granted for a particular request.</p>
+
+              <section id="effective-acl-resource" inlist="" rel="schema:hasPart" resource="#effective-acl-resource">
+                <h3 property="schema:name">Effective ACL Resource</h3>
```

Comment:
I think the paragraph before it gives a hint that no Authorization may be matched. There is also the bit in #authorization-representation about it being RDF and can contain anything - implying it can be empty or random junk. Lastly, the point of #effective-acl-resource is just to find the right acl-resource. #authorization-conformance , #authorization-matching detail the rest.

Reply:
I think the fact that a resource might end up without Authorization is important enough to spell out loud. Especially because it is unclear to me what the behaviour of the server becomes in that case.

Say my new resource `x` gets initialized with a bogus effective ACL resource `y`.

Then, in theory, no one has access to `x`, furthermore, no one can even discover `y` to fix that.

I don't think it's impossible to mitigate, but I'm not sure which issues address the problem and I'd like to see them (or one created) and linked to this short sentence for later addressing it.

Reply:
#authorization-evaluation says:

>Evaluation stops when all access permission requests have been granted by one or more Authorizations. Authorizations that do not match a required access permission have no effect on the outcome of the evaluation. Access is granted when conforming Authorizations are matched, otherwise access is denied.

>Say my new resource x gets initialized with a bogus effective ACL resource y.

Let's say that. What kind of "conforming" application is creating a bogus ACL resource? Or what kind of conforming server is perhaps initialising a bogus ACL resource?

>Then, in theory, no one has access to x, furthermore, no one can even discover y to fix that.

True except #loss-of-control-mitigation says:
>It is assumed that systems using WAC have mechanisms to mitigate loss of control of the resources, such as by ensuring valid ACL resources and Authorizations, by providing administrative functions on the storage, or by implicitly granting control over all resources in a storage to owners.

Reply:
> True except #loss-of-control-mitigation says:

I missed that bit.

Reply:
PS: Thanks for that last bit of explanation @csarven and all the last minute changes. I'm more than happy to merge in the current state!